tests/contrib/django/django_tests.py

```diff
@@ -706,7 +709,10 @@ def test_request_metrics_301_prepend_www(self):
 
         # enable middleware wrapping
         client = get_client()
-        client.instrument_django_middleware = True
+        client.config.instrument_django_middleware = True
+
+        from elasticapm.contrib.django.middleware import TracingMiddleware
+        TracingMiddleware._elasticapm_instrumented = False
```

Comment:
why is this set to `false`?

Reply:
we're resetting it in case another test already instrumented the middleware. Python modules and classes are basically global state, so we need to ensure that the global state is correct before running the test